src/app/icons/icons.component.ts

```diff
@@ -1,10 +1,18 @@
 import {
-    Component, AfterViewInit, OnDestroy, Renderer2, ElementRef, ChangeDetectorRef
+    Component,
```

Comment:
At some point do you think we should run `clang:format` or [https://prettier.io/](prettier.io) on the site code?
Maybe after it settles down from all the changes.

Reply:
I did not because that would cause major changes that distract from the real changes. When we merge the website into master we'll have to do that though.

Reply:
Yes, that's what I meant. Not for this change but something to consider in the future.